src/components/FirmwareUpdate/index.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useEffect, useCallback, useReducer } from "react";
+import { useTranslation } from "react-i18next";
+import { NativeModules } from "react-native";
+import { useSelector, useDispatch } from "react-redux";
+import { Device } from "@ledgerhq/live-common/lib/hw/actions/types";
+import { Button, Icons } from "@ledgerhq/native-ui";
+import {
+  BackgroundEvent,
+  nextBackgroundEventSelector,
+} from "../../reducers/appstate";
+import {
+  clearBackgroundEvents,
+  dequeueBackgroundEvent,
+} from "../../actions/appstate";
+import BottomModal from "../BottomModal";
+import GenericErrorView from "../GenericErrorView";
+import { DeviceInfo } from "@ledgerhq/live-common/lib/types/manager";
+import useLatestFirmware from "../../hooks/useLatestFirmware";
+import ConfirmRecoveryStep from "./ConfirmRecoveryStep";
+import FlashMcuStep from "./FlashMcuStep";
+import FirmwareUpdatedStep from "./FirmwareUpdatedStep";
+import ConfirmPinStep from "./ConfirmPinStep";
+import ConfirmUpdateStep from "./ConfirmUpdateStep";
+import DownloadingUpdateStep from "./DownloadingUpdateStep";
+import { track } from "../../analytics";
+
+type Props = {
+  device: Device;
+  deviceInfo: DeviceInfo;
+  isOpen: boolean;
+  onClose: (restoreApps?: boolean) => void;
+};
+
+const BluetoothNotSupportedError: Error = {
```

Comment:
If this is a new error it should live in live-common https://github.com/LedgerHQ/ledger-live-common/blob/develop/src/errors.ts please update the live-common pr (or create one if needed) to get that from there 🙏🏼

Reply:
Done